src/components/storyUtils/ListShowcase.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import List from '../List';
+import { ListProps } from '../List/List';
+import { SelectOption } from '../Select/Select';
+
+interface Props extends Omit<ListProps, 'data'> {
+  itemsCount: number;
+}
+
+const ListShowcase: React.FC<Props> = ({ itemsCount, rowSize, width, height, isVirtualized }) => {
+  const items: SelectOption[] = Array(itemsCount)
+    .fill({})
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    .map((_, i) => {
```

Comment:
double `__` underscore is for muting the no unused vars so you can remove this and simplify the next line to one liner